doc/kuksa_analysis/kuksa_requirementss.md

```diff
@@ -0,0 +1,202 @@
+            - "not provided": The signal is basically known, but currently no provider is serving it or the provider lost the connection to the backend component serving this signal.
+            - "invalid": The signal is being served by a provider, but currently has no valid value. This may happen if a specific value cannot be determined because of some (temporary) physical sensor error or a target value is currently not known, e.g. if a window is moved by a passenger via up/down button.
+       
+      2.  The Signal Consumer only shall get values for signal to which it has permission to.
+      3.  The Signal Consumer shall provide the paths of the requested signals, which optionally may contain wildcards.
+  
+### FR3-ASC
+    Title: Subscription and Notification for Signal Value Changes
+
+    Status: 🟡
+
+    Description: The Signal Consumer shall subscribe to sensor values or actuator current values of a signal and then get notified when some signal value changed.
+      *  IF there is an error
+          THEN the Signal Consumer shall not receive any signal value, just one single error with detailed error information
+          ELSE the Signal Consumer shall receive exactly all the signals’ states(failure or value) requested.
+      *  The Signal Consumer only shall get values for signal to which it has permission to.
+      *  The Signal Consumer shall provide the paths of the requested signals.
+
+### FR4-ASC
+    Title: Filtered Interval Subscription for Signal Values
+
+    Status: 🟡
+
+    Description: The Signal Consumer shall subscribe and be able to apply a filter to receive a signal values with an interval of x milliseconds.
+      *  IF there is an error
+          THEN the Signal Consumer shall not receive any signal value, just one single error with detailed error information
+          ELSE the Signal Consumer shall receive exactly all the signals’ states(failure or value) requested.
+      *  The Signal Consumer only shall get values for signal to which it has permission to.
+      *  The Signal Consumer shall provide the paths of the requested signals.
+
+### FR5-ASC
+    Title: Accessing Static Metadata of Signals
+
+    Status: 🟡
+
+    Description: A Signal Consumer shall be able to get static Metadata from signals.
+    - Details:
+      * All sensors, actuators, and attributes values for which a Signal Consumer has permission to.
+      * The Signal Consumer shall provide the path, paths or wildcard of the signals.
+
+### FR6-ASC
+    Title: Retrieving Attribute Values for Signal Consumption
+
+    Status: 🟡
+
+    Description: A Signal Consumer shall be able to get attributes values.
+
+### FR7-ASC
+    Title: Actuation of Actuator Value with Databroker Confirmation
+
+    Status: 🟡
+
+    Description: The Signal Consumer shall be able to actuate a value of an actuator. This value has to be forwarded to the provider of this actuator if the provider is available, otherwise the target value is lost:
```

Comment:
```suggestion
    Description: The Signal Consumer shall be able to request an actuation. This request has to be forwarded to the provider of this actuator if the provider is available, otherwise the request gets lost:
```